crux-mir/test/conc_eval/intTest/test1129.rs

```diff
@@ -0,0 +1,12 @@
+#[cfg_attr(crux, crux::test)]
```

Comment:
```suggestion
//! Regression test for missing override for the `rotate_left` intrinsic

#[cfg_attr(crux, crux::test)]
```
Or something like that, so the reader doesn't have to look up #1129 to understand what this test is for